fabric/src/main/java/dev/architectury/hooks/level/entity/fabric/PlayerHooksImpl.java

```diff
@@ -23,6 +23,6 @@
 
 public class PlayerHooksImpl {
     public static boolean isFake(Player player) {
-        return false;
+        return !FakePlayers.EVENT.invoker().isFakePlayer(player).isTrue();
```

Comment:
1. The logic seems a bit backwards on this: A player should be considered fake if this event handler returns `false`?
2. As outlined in [this comment](https://github.com/architectury/architectury-api/issues/261#issuecomment-1133887562), I feel like a reasonable default if no boolean result is given (i.e. `pass`) should be to check whether the player extends `ServerPlayer`, but themselves *isn't* a ServerPlayer